internal/v1/api.yaml

```diff
@@ -1262,6 +1267,39 @@ components:
             Array of image requests. Having more image requests in a single blueprint is currently not supported.
         customizations:
           $ref: '#/components/schemas/Customizations'
+    BlueprintExportResponse:
+      required:
+        - id
+        - name
+        - description
+        - distribution
+        - customizations
+        - metadata
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        description:
+          type: string
+        distribution:
+          $ref: '#/components/schemas/Distributions'
+        customizations:
+          $ref: '#/components/schemas/Customizations'
+        metadata:
+          $ref: '#/components/schemas/BlueprintMetadata'
+    BlueprintMetadata:
+      required:
+        - parent_id
+        - exported_at
+      properties:
+        parent_id:
+          type: string
+          format: uuid
+          nullable: true
+        exported_at:
+          type: string
```

Comment:
Consider creating two root elements: metadata + data so the root namespace is nice and clean. But this works too.

Reply:
Thinking about this comment. Do you like the idea of having the metadata + data only for the exported blueprint? So that we export it like that? Then there is a thought of mine that we do not even have to export the metadata at all, as it does not really tell users that much.
Or would you like to see the "data" element even in the case of the CreateBlueprintRequest, where we have the metadata as well? That would be a big change of the API.

Reply:
My thinking is if we are going to create a new endpoint `xxx/export` then we can design whatever we want and I was thinking to create a "clean" root namespace by separating it into metadata part and the actual data. Now, I have weak arguments why to do it other than it creates a "clean" root namespace, it is just an idea. What you have works just fine, as long as "metadata" is not added into the blueprint itself.

If course if you want to use the Content-Type approach, then it does not make any sense and my comment is irrelevant in that case. Apologies for the confusion there.

Reply:
Okay I take my comment back, I haven’t realized metadata are going to be stored alongside the data for recording purposes. This is fine, what I propose does worsen things actually.